### PR TITLE
feat: reporters option, default to progress

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ const getDefaults = () => {
     browserstackLaunchers: Object.assign(browserstackLaunchers),
     showQUnitUI: inServerMode(),
     preferHeadless: true,
+    reporters: [ isCI ? 'dots' : 'progress', 'coverage'],
```
Comment (brandonocasey) [inline index.js:123]: perhaps we should add 'coverage' after user settings if `coverage` is true? That way the user won't have to worry about manually adding the coverage reporter.

**Comment by brandonocasey** [comment]
I also wonder if we want to use `dots` in `serverMode` but `progress` everywhere else since we will be able to see progress in the ui for `serverMode` but will not for everything else.

File: index.js (hunk 1 above)
Comment (gkatsev) [reply index.js:123]: Eh, if someone is already making changes they can do the extra step of adding coverage if they need it.

**Comment by gkatsev** [comment]
Yeah, using dots in serverMode might be make sense. Let me try it out and I'll update this.

**Comment by gkatsev** [comment]
Here's a comparison of progress vs dots in server mode:
progress:
Screen Shot 2021-01-14 at 15 10 56
dots:
Screen Shot 2021-01-14 at 15 14 01

progress works ok for this I think but if you think dots is better I can make dots default for server mode.

**Review by brandonocasey — APPROVED** [review]
Progress looks different than I expected, I think this change makes sense to me now with the visual aid.

**Pull request merged**